demisto_sdk/commands/content_graph/common.py

```diff
@@ -270,15 +272,27 @@ def by_schema(path: Path) -> "ContentType":
         raise ValueError(f"Could not find content type in path {path}")
 
 
+class Relationship(BaseModel):
+    relationship: Optional[RelationshipType] = None
+    source_id: Optional[str] = None
+    source_type: Optional[ContentType] = None
+    source_fromversion: Optional[str] = None
+    source_marketplaces: Optional[List[MarketplaceVersions]]
+    target: Optional[str] = None
+    target_type: Optional[ContentType] = None
+    mandatorily: Optional[bool] = None
+
+
 class Relationships(dict):
     def add(self, relationship: RelationshipType, **kwargs):
         if relationship not in self.keys():
             self.__setitem__(relationship, [])
-        self.__getitem__(relationship).append(kwargs)
+        self.__getitem__(relationship).append(Relationship.parse_obj(kwargs).dict())
```

Comment:
i think you should use `.dict(exclue_none=True)` so null values will not be parsed.

Reply:
FIixed